alibi/explainers/ale.py

```diff
@@ -38,39 +38,39 @@ def __init__(self,
         Parameters
         ----------
         predictor
-            A callable that takes in an NxF array as input and outputs an NxT array (N - number of
-            data points, F - number of features, T - number of outputs/targets (e.g. 1 for single output
-            regression, >=2 for classification).
+            A callable that takes in an `N x F` array as input and outputs an `N x T` array (`N` - number of
+            data points, `F` - number of features, `T` - number of outputs/targets (e.g. 1 for single output
+            regression, >=2 for classification)).
         feature_names
             A list of feature names used for displaying results.
         target_names
             A list of target/output names used for displaying results.
         check_feature_resolution
-            If true, the number of unique values is calculated for each feature and if it is less than
-            `low_resolution_threshold` then the feature values are used for gridpoints instead of quantiles.
+            If ``True``, the number of unique values is calculated for each feature and if it is less than
```

Comment:
This is interesting, I wasn't aware single backticks generate italic text and double backticks generate the red text in boxes.

For style, I'm thinking whether we should also use double backticks for references to parameters from other parameter definitions, e.g.
```
Only used if ``check_feature_resolution=True``. 
```
The it seems to me more clear which parameter has which effect on other parameters by just glancing at the API docs as references to parameter names would be highlighted.

@ascillitoe